src/go/rpk/pkg/api/admin/admin.go

```diff
@@ -241,13 +344,19 @@ func (a *AdminAPI) getURLFromBrokerID(brokerID int) (string, bool) {
 }
 
 // sendOne sends a request with sendAndReceive and unmarshals the body into
-// into, which is expected to be a pointer to a struct.
-func (a *AdminAPI) sendOne(method, path string, body, into interface{}) error {
+// into, which is expected to be a pointer to a struct
+//
+// Set `retryable` to true if the API endpoint might have transient errors, such
+// as temporarily having no leader for a raft group.  Set it to false if the endpoint
```

Comment:
Where can we rpk maintainers / devs determine whether we should use true or false here? Ping core, or is this in some swagger docs, or..?

Reply:
Yes, the answer today is to ask core and/or read the source code.  Admin API docs are on the docs team's queue.